new/src/ReportFilter.test.ts

```diff
@@ -0,0 +1,268 @@
+import { repository } from "./repository";
```

Comment:
It is good to have at least some tests. But we should increase our test coverage for all of the added features. Maybe not in this PR, but we should not forget about it